examples/stimuli/tracker_dealer.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+"""
+======================
+Tracker Dealer Example
+======================
+
+This file shows how to interleave multiple Tracker objects using
+:class:`expyfun.stimuli.TrackerDealer`.
+
+In this case, a modeled human subject generates two curves (one for each trial
+type: 1 & 2)
+
+@author: maddycapp27
+"""
+
+import numpy as np
+from expyfun.stimuli import TrackerUD, TrackerDealer
+from expyfun.analyze import sigmoid
+import matplotlib.pyplot as plt
+
+# define parameters of modeled subject (using sigmoid probability)

```

Comment:
Thanks, I'll definitely keep that in mind for the future! For this example, I agree that it's not very necessary though.